/jobfs/local/pbs/mom_priv/jobs/8043177.r-man2.SC: line 3: 
File to test various parts of fio.py and reprocess.py
: command not found
import: unable to open X server `' @ error/import.c/ImportImageCommand/368.
import: unable to open X server `' @ error/import.c/ImportImageCommand/368.
import: unable to open X server `' @ error/import.c/ImportImageCommand/368.
/jobfs/local/pbs/mom_priv/jobs/8043177.r-man2.SC: line 15: from: command not found
/jobfs/local/pbs/mom_priv/jobs/8043177.r-man2.SC: line 17: from: command not found
/jobfs/local/pbs/mom_priv/jobs/8043177.r-man2.SC: line 19: from: command not found
import: unable to open X server `' @ error/import.c/ImportImageCommand/368.
/jobfs/local/pbs/mom_priv/jobs/8043177.r-man2.SC: line 21: from: command not found
import: unable to open X server `' @ error/import.c/ImportImageCommand/368.
import: unable to open X server `' @ error/import.c/ImportImageCommand/368.
/jobfs/local/pbs/mom_priv/jobs/8043177.r-man2.SC: line 31: syntax error near unexpected token `('
/jobfs/local/pbs/mom_priv/jobs/8043177.r-man2.SC: line 31: `def regularbounds(x,fix=False):'
